stdlib/strings.tau

```diff
@@ -0,0 +1,169 @@
+Clone = fn(str) {
+	ret = ""
+
+	for i = 0; i < len(str); ++i {
+		ret += str[i]
+	}
+
+	return ret
+}
```

Comment:
This _feels_ inefficient to me. I _feel_ like there should be a way to clone an object in Tau? 🤔 cc @NicoNex

Reply:
I thought about that and checked how Python does it and I believe we don't need the `Clone` after all since the strings are immutable.

Additionally you could do the cloning simply with `cloned = "" + mystring`